docs/getting_started/dashboard/dashboard_block_reference.md

```diff
@@ -0,0 +1,168 @@
+
+!!! note
+
+    Before you add a block that involves products, product types, or product categories, make
+    sure your that your [user Role](permissions_and_users.md) has
+    the `Product/View` and `Product type/View` permission.
+
+## Common content block
+
+Displays a list of content items created by all members of the organization that the user belongs to.
+It shows the following tabs: **Content**, **Scheduled**, **Media**.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of content and submit your changes.
+
+## Ibexa News block
+
+Presents a list of recent blog posts or articles published at `ibexa.co` blog (product related).
+It includes title, image, publication date, and link to article details.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **News limit** - Set a maximum number of news to be displayed. Default = 7, minimum value = 1, and maximum = 10.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of news and submit your changes.
+
+## My content block
+
+Displays a list of content items created by the user who is currently logged in.
+It shows the following tabs: **Drafts**, **Scheduled**, **Content**, **Media**, **Drafts to review**.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of content and submit your changes.
+
+## Orders by status block [[% include 'snippets/commerce_badge.md' %]]
+
+Displays a chart presenting orders split by status, with their number and percentage.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Statuses** - Set the statuses of orders that should be included in the list. Default value = All. The list of statuses depends on configured [order workflow](https://doc.ibexa.co/en/latest/commerce/order_management/configure_order_management/) and can be customized.
+- **Time period** - Set the time period: All time, Last day, Last week, Last month. Default value = Last month.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of orders and submit your changes.
+
+## Products by category block [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+Displays a chart presenting the products split by category, together with the percentage.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Number of categories** - Set a maximum number of categories to be displayed. Default value = 3, minimum value = 1, and maximum = 20.
```

Comment:
```suggestion
- **Number of categories** - Set a maximum number of categories to be displayed. Default value = 10, 
```
As I checked in nightly, this limit is not implemented in backend. You can enter different value.
Currently default value is 10 and I would keep it (better than 3).